test/selenium/app_test.js

```diff
@@ -3,18 +3,15 @@
 var Prepper = require('./lib/testPrepperSelenium.js');
 var webdriver = require('selenium-webdriver');
 
-var HOST_URL = 'https://fxbox.github.io/app';
+var HOST_URL = 'http://fxbox.github.io/app';
```

Comment:
We run foxbox with `--disable-tls`, so it's preferable to not mix http with https. We also cannot go with full https because we have to manually add one certificate exception